### PR TITLE
Add new app and v2 compatible Figlet component to the readme

### PR DESCRIPTION
**Pull request by hexrcs**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ Feel free to play around with the code and fork this repl at [https://repl.it/@v
 - [gomoku-terminal](https://github.com/acrazing/gomoku-terminal) - Play online Gomoku in the terminal.
 - [cfpush](https://github.com/mamachanko/cfpush) - An interactive Cloud Foundry tutorial in your terminal.
 - [startd](https://github.com/mgrip/startd) - Turn your React component into a web app from the command-line.
+- [wiki-cli](https://github.com/hexrcs/wiki-cli) - Search Wikipedia and read summaries directly in your terminal.
```
Comment (sindresorhus) [inline readme.md]: I think you should spend some more time on your readme there. The point of this list is to show off Ink usage, and you don't even have a screenshot (or better yet, animated GIF) of how it looks.

File: readme.md [hunk 2]
```diff
@@ -985,6 +986,7 @@ Similar to `useApp`, it's equivalent to consuming `StdoutContext` directly.
 - [ink-divider](https://github.com/JureSotosek/ink-divider) - A divider component.
 - [ink-progress-bar](https://github.com/brigand/ink-progress-bar) - Configurable component for rendering progress bars.
 - [ink-table](https://github.com/maticzav/ink-table) - Table component.
+- [ink-ascii](https://github.com/hexrcs/ink-ascii) - Awesome text component with more font choices, based on Figlet.
```
Comment (sindresorhus) [inline readme.md]: This needs tests.

File: readme.md (hunk 1 above)
Comment (hexrcs) [reply readme.md]: Will do! Thanks for the tips.

File: readme.md (hunk 2 above)
Comment (hexrcs) [reply readme.md]: Will do 👍

Comment (LitoMore) [inline readme.md]: Have you tested your package on production?

You set `main: "index.js"` in your [package.json](https://github.com/hexrcs/ink-ascii/blob/master/package.json#L5) but your main file is `dist/index.js`.

Comment (hexrcs) [reply readme.md]: Sorry about that, I'm very new to library authoring. Thanks for pointing it out and fixing it!

**Comment by sindresorhus** [comment]
Any progress here?

**Comment by hexrcs** [comment]
@sindresorhus Sorry for the delay. Working on it, expecting to have it done tomorrow.

**Comment by hexrcs** [comment]
@sindresorhus Added tests for `ink-ascii` and a screen recording gif to `wiki-cli`.

**Comment by LitoMore** [comment]
@hexrcs Could you also fix the merge conflict?

**Comment by hexrcs** [comment]
Done!

**Pull request merged**